test/restv2_test.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+import urllib3
+from mocket.plugins.httpretty import HTTPretty, httprettified
+
+HTTPRETTY = HTTPretty
+
+# Disable HTTPS warnings because all requests are mocked
+urllib3.disable_warnings()
+
+from bitfinex.rest.restv2 import Client
+
+class BitfinexTest(unittest.TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_instantiate_client(self):
+        """test instantiate client"""
+        self.assertIsInstance(self.client, Client)
+
+    @httprettified
+    def testm1_platform_status(self):
+        """test method 1/34 platform_status"""
+        mock_body = '[1]'
+        url = "https://*/*"
+        HTTPRETTY.register_uri(HTTPRETTY.GET, url, body=mock_body, status=200)
+        platform_status = self.client.platform_status()
+        self.assertEqual([1], platform_status)
+
+
+    @httprettified
+    def testm2_shoud_have_tickers(self):
+        """test method 2/34 tickers"""
+        mock_body = "[['tIOTBTC', 0.00012012, 207246.56609548, 0.00012048, 230371.29034138, -8.28e-06, -0.0644, 0.00012035, 3215468.26026294, 0.00012863, 0.00011545], ['tIOTUSD', 0.95969, 24306.55703175, 0.96214, 60186.47595187, -0.03534, -0.0355, 0.95925, 13274504.88034301, 0.99459, 0.90661]]"
+        url = "https://*/*"
+        HTTPRETTY.register_uri(HTTPRETTY.GET, url, body=mock_body, status=200)
+        tickers_response = self.client.tickers(['tIOTBTC', 'tIOTUSD'])
+        self.assertIsInstance(tickers_response, list)
+
+    @httprettified
+    def testm3_shoud_have_ticker(self):
+        """test method 3/34 ticker"""
+        mock_body = "[0.00011989, 224267.1563644, 0.00012015, 265388.46405117, -8.12e-06, -0.0634, 0.00012, 3206446.72556849, 0.00012812, 0.00011545]"
+        url = "https://*/*"
```

Comment:
This does not seem to work. It might be because of how python-mocker works with httppretty. However for now, spesify the exact url that you want to mock.

```python
    @httprettified
    def testm3_shoud_have_ticker(self):
        """test method 3/34 ticker"""
        mock_body = "[0.00011989, 224267.1563644, 0.00012015, 265388.46405117, -8.12e-06, -0.0634, 0.00012, 3206446.72556849, 0.00012812, 0.00011545]"
        url = "https://api.bitfinex.com/v2/ticker/tIOTBTC"
        HTTPRETTY.register_uri(HTTPRETTY.GET, url, body=mock_body, status=200)
        ticker_response = self.client.ticker('tIOTBTC')
        self.assertIsInstance(ticker_response, list)
```